pywt/data/_readers.py

```diff
@@ -137,3 +137,43 @@ def ecg():
     fname = os.path.join(os.path.dirname(__file__), 'ecg.npy')
     ecg = np.load(fname)
     return ecg
+
+
+def nino():
+    """
+    NINO3 sea surface temperature 1871-1996 (in C; from the UKMO GISST2.3):
```

Comment:
insert a blank line here. ``autodoc`` formatting makes tables and should only grab the summary line.